### DISCUSSION TITLE
The accuracy of longitude is lost at the high zoom level

### DISCUSSION THREAD

**Discussion by marvinluo1** [visgl/deck.gl#8367] [Q&A] [answered]
I have a continuous list of latitude and longitude coordinates that are close together, approximately 10cm apart. I want to visualize them on a map using the `ScatterplotLayer` layer.
```
[
  {lat: -34.00310207458188,lng: 60.0018187129627},
  {lat: -34.00310262293301,lng: 60.00181957125977},
  {lat: -34.00310317016578,lng: 60.00182043059479},
  {lat: -34.003103716278844,lng: 60.001821290965665},
  {lat: -34.00310426127086,lng: 60.001822152370245},
  {lat: -34.00310480514049,lng: 60.00182301480641},
  {lat: -34.003105347886375,lng: 60.001823878272035},
  {lat: -34.00310588950719,lng: 60.001824742764995},
  {lat: -34.003106430001594,lng: 60.00182560828315},
  {lat: -34.003106969368254,lng: 60.00182647482438}
]
```

Expected:
<img width="300" alt="image" src="https://github.com/visgl/deck.gl/assets/6326386/20aeb95f-eaef-4087-9713-a21a09c53a6a">

Actually:
<img width="300" alt="image" src="https://github.com/visgl/deck.gl/assets/6326386/5bd4a2f3-8b67-4bb7-a589-e2ff178a75e1">

What I tried: 
1. I attempted to add the line `extensions: [new Fp64Extension()]` to my code. However, this caused the content not to be displayed.

2. In a fortunate attempt, I updated the longitude values from 60.xxx to 0.xxx. This resulted in the coordinates being rendered correctly on the map. Interestingly, I observed that as the longitude values increased, more longitudes were drawn at the same location.

<img width="350" alt="image" src="https://github.com/visgl/deck.gl/assets/6326386/2b6df4a5-4aaf-46c1-8d58-4a72f56e4b2f">

<img width="400" alt="image" src="https://github.com/visgl/deck.gl/assets/6326386/b6c71491-07cb-40ec-8f6b-7e1376c30578">

I have included the demo code below.([preview in codepen](https://codepen.io/codepenrnd/pen/oNVgdLW)):
```
/*
 * https://deck.gl/docs/api-reference/layers/scatterplot-layer
 */
const { DeckGL, ScatterplotLayer, COORDINATE_SYSTEM, Fp64Extension } = deck;

const layer = new ScatterplotLayer({
  id: "ScatterplotLayer",
  data: [
    {lat: -34.00310207458188,lng: 60.0018187129627},
    {lat: -34.00310262293301,lng: 60.00181957125977},
    {lat: -34.00310317016578,lng: 60.00182043059479},
    {lat: -34.003103716278844,lng: 60.001821290965665},
    {lat: -34.00310426127086,lng: 60.001822152370245},
    {lat: -34.00310480514049,lng: 60.00182301480641},
    {lat: -34.003105347886375,lng: 60.001823878272035},
    {lat: -34.00310588950719,lng: 60.001824742764995},
    {lat: -34.003106430001594,lng: 60.00182560828315},
    {lat: -34.003106969368254,lng: 60.00182647482438}
  ],

  /* props from ScatterplotLayer class */
  filled: true,
  getFillColor: [255, 0, 0],
  getLineColor: [255, 0, 0],
  getPosition: (d) => [d.lng, d.lat],
  // extensions: [new Fp64Extension()],
  getRadius: (d) => 0.1,
  lineWidthMinPixels: 0.1,
  radiusMaxPixels: 10,
  radiusMinPixels: 10,
  radiusScale: 1,
  opacity: 0.8,
  pickable: true
});

new DeckGL({
  initialViewState: {
    longitude: 60.00182647482438,
    latitude: -34.003106969368254,
    zoom: 25,
    maxZoom: 25
  },
  controller: true,
  getTooltip: ({ object }) => object && `${object.lat}, ${object.lng}`,
  layers: [layer]
});

```

I would appreciate any advice or guidance you can provide. Am I using deck.gl incorrectly, or is this possibly a bug? Thank you for your assistance.

**Comment by Pessimistress** [comment, marked as answer]
#8235 ?

**Reply by marvinluo1** [reply]
That's really the reason, Because it is still in the development process, I only tested it on this machine, but I have not found this problem. Thank you very much.
